src/assets/i18n/en_US.json

```diff
@@ -2806,5 +2812,11 @@
   "account.user.new.OrderLimit.error.valid": "Please enter a valid money value (limit per order).",
   "account.user.new.Budget.error.valid": "Please enter a valid money value (Budget).",
   "account.user.new.roles.dialog.title": "Roles",
-  "account.user.new.roles.dialog.action.ok": "OK"
+  "account.user.new.roles.dialog.action.ok": "OK",
+  "account.organization.user_management": "User Management",
+  "account.organization.user_management.user_detail": "User Detail",
```

Comment:
why isn`t this (existing) key used here? 
account.user.details.heading": "User Details

is user detail or user details right? pls, use always the same pattern, e.g. in quotes it is details